x-pack/plugins/upgrade_assistant/public/components/tabs/checkup/deprecations/reindex/flyout/progress.test.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import { ReindexStatus, ReindexStep } from '../../../../../../../common/types';
+import { ReindexProgress } from './progress';
+
+describe('ReindexProgress', () => {
+  it('renders', () => {
+    const wrapper = shallow(
+      <ReindexProgress
+        lastCompletedStep={ReindexStep.created}
+        reindexStatus={ReindexStatus.inProgress}
+        reindexTaskPercComplete={null}
+        errorMessage={null}
+      />
+    );
+
+    expect(wrapper).toMatchInlineSnapshot(`
+<Component
+  steps={
+    Array [
+      Object {
+        "status": "incomplete",
+        "title": "Setting old index to read-only",
+      },
+      Object {
+        "status": "incomplete",
+        "title": "Creating new index",
+      },
+      Object {
+        "status": "incomplete",
+        "title": "Reindexing documents",
+      },
+      Object {
+        "status": "incomplete",
+        "title": "Creating alias",
```

Comment:
Thoughts on changing the wording here? I think it's less about creating the alias and more about swapping it out with an alias.

Maybe: `Swapping index with an alias`